### PR TITLE
Update README to reflect changes in this iteration

### PR DESCRIPTION
**Pull request by laugengebaeck**
## What does this change?

Updates the readme according to the changes made in this semester.

## Checklist

- [x] Your branch has no merge conflicts with master
- [x] All related commits are squashed together

### CODE REVIEW

**Comment by laugengebaeck** [comment]
Would be nice if someone wants to have a look at this. But if you want to merge, please remember to do **squash and merge** (especially you, Johann ;)

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # IMAPClient [![Coverage Status](https://coveralls.io/repos/github/hpi-swa-teaching/IMAPClient/badge.svg?branch=develop)](https://coveralls.io/github/hpi-swa-teaching/IMAPClient?branch=develop)
  
-Welcome to our IMAP Client project. We are a team of 4th semester IT-Systems Engineering students and are happy to take over this project for the sixth iteration.
+Welcome to our IMAP Client project. We are a team of 4th semester IT-Systems Engineering students and are happy to take over this project for the seventh iteration.
```
Comment (elenagensch) [inline README.md:3]: are we already seventh itertaion or does this refer to the next team, this is a bit confusing

Comment (laugengebaeck) [reply README.md:3]: This refers to the next team 😅 The team before us also updated it that way, so I did too.

Comment (elenagensch) [reply README.md:3]: I'm pretty sure, someone from our team updated the Readme this year (our names were added), but ok, it can stay that way

**Pull request merged**